pkg/rebuild/maven/infer.go

```diff
@@ -125,8 +125,9 @@ func (Rebuilder) InferStrategy(ctx context.Context, t rebuild.Target, mux rebuil
 	if err != nil {
 		return cfg, errors.Wrap(err, "fetching JDK")
 	}
-	if jdk == "" {
-		return cfg, errors.New("no JDK found")
+	if jdk == "" || JDKDownloadURLs[jdk] == "" {
+		log.Print("falling back to JDK 11")
+		jdk = "11"
```

Comment:
@msuozzo @wbxyz Don't you think this is more "hidden" for a default to be configured?

Reply:
I might turn the log message into a comment. (if we were to keep the log message, I'd probably want to split the cases so you can give more specific feedback to differentiate the cases 1. the jdk was empty 2. the jdk had no associated url).